bundles/org.pitest.pitclipse.launch/src/org/pitest/pitclipse/launch/config/PackageFinder.java

```diff
@@ -75,49 +75,49 @@ public boolean visit(IResourceProxy proxy) throws CoreException {
         }
 
         private Set<String> getPackagesFromRoot(IJavaProject project, String handleId) throws CoreException {
-            Builder<String> builder = ImmutableSet.builder();
+            Builder<String> setBuilder = ImmutableSet.builder();
```

Comment:
IHMO `setBuilder` doesn't look really clearer than just `builder` — I have actually been confused by the "set" prefix because I first read as a verb. I think `builder` is ok because the function is pretty short, but if you do want a more explicit name what about something like `packagesBuilder`, or `packagesSetBuilder`?

Reply:
I changed the name since it had the same name as the field and SonarQube was correctly complaining.
Those names will be changed anyway when getting rid of Guava, I've already started to work on that in https://github.com/pitest/pitclipse/tree/task_136-Get_rid_of_Guava
At that point, they will probably be simply called `packages`, etc.

Reply:
Alright!